digicampipe/utils/disp.py

```diff
@@ -0,0 +1,290 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+from tqdm import trange
+
+
+def disp_eval(A, width, length, cog_x, cog_y,
+              x_offset, y_offset, psi, skewness,
+              size, leakage2, method):
+
+    parvals = A.valuesdict()
```

Comment:
Could you rename `A` to an actual noun please ? I think it makes it clearer what it is

Reply:
I would say the same for `parvals`